test/e2e/examples_test.go

```diff
@@ -21,9 +21,56 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func TestXonoticGameServerReady(t *testing.T) {
+	t.Parallel()
+	gs := &agonesv1.GameServer{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "xonotic-",
+		},
+
+		Spec: agonesv1.GameServerSpec{
+			Container: "xonotic",
+			Ports: []agonesv1.GameServerPort{{
+				ContainerPort: 26000,
+				Name:          "default",
+				PortPolicy:    agonesv1.Dynamic,
+				Protocol:      corev1.ProtocolUDP,
+			}},
+			Health: agonesv1.Health{
+				InitialDelaySeconds: 300,
+				PeriodSeconds:       60,
```

Comment:
We can pull this down to something a bit more reasonable....

```suggestion
				InitialDelaySeconds: 60,
				PeriodSeconds:       5,
```

(The health ping should happen every 2 seconds).

But otherwise this looks good!